4-projects/modules/base_env/example_peering_project.tf

```diff
@@ -261,3 +233,110 @@ resource "google_compute_firewall" "allow_lb" {
 
   target_tags = ["allow-lb"]
 }
+
+// Allow SSH via IAP when using the Firewall Secure Tags.
+resource "google_compute_network_firewall_policy" "allow_iap_firewall_policy" {
+  count = var.optional_iap_fw_rules_enabled ? 1 : 0
+
+  name    = "fp-${local.env_code}-allow-iap-policy"
+  project = module.peering_project.project_id
+}
+
+resource "google_compute_network_firewall_policy_rule" "allow_iap_ssh" {
```

Comment:
is there a module for this type of firewall rule?

Reply:
[PR 514](https://github.com/terraform-google-modules/terraform-google-network/pull/514) will fix the issue.

Reply:
@Samir-Cit since network firewall policy module is fixed is it possible to call the module instead of resource?